docs/guide/shop_templates/shop_twig_functions.md

```diff
@@ -1,126 +1,69 @@
-# Twig extension [[% include 'snippets/commerce_badge.md' %]]
-
-[[= product_name_com =]] offers the following Twig functions and filter for use in templates:
+# Shop-related Twig functions
 
 ## Functions
 
 |Name|Parameters|Description|
 |--- |--- |--- |
-|`ses_render_field`|`CatalogElement $catalogElement`</br>`string $fieldIdentifier = ''`</br>`array $params = array()`|Renders a CatalogElement containing a Field|
-|`ses_navigation`|`array $params = array()`|Returns navigation node tree|
+|`ses_render_field`|`CatalogElement $catalogElement`</br>`string $fieldIdentifier = ''`</br>`array $params = array()`|Renders a Field of a CatalogElement|
 |`ses_product`|`array $params = array()`|Returns a product node</br>`{% set product = ses_product({'sku': line.sku, 'variantCode': line.variantCode }) %}`|
 |`ses_variant_product_by_sku`|`string $sku`|Fetches a VariantProductNode for the given SKU|
+ses_assets_by_group
```

Comment:
Should this function name be followed by an example and a description?

Reply:
Unfortunate leftover from my notes ;)